packages/components/psammead-image/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Psammead Image Changelog
 
-| Version | Description |
-|---------|-------------|
-| 0.2.1   | [PR#231](https://github.com/BBC-News/psammead/pull/231) Add link to Storybook to README |
-| 0.2.0 | [PR#234](https://github.com/BBC-News/psammead/pull/234) Change height prop to be optional. Update the storybook example. |
-| 0.1.1 | [PR#230](https://github.com/BBC-News/psammead/pull/230) Add README to Storybook |
-| 0.1.0 | [PR#225](https://github.com/BBC-News/psammead/pull/225) Create initial package, pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
+| Version | Description                                                                                                                                    |
+| ------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Prettier auto-adds this, it's a shame as it makes it much more difficult to maintain!

Whilst we think of a more long term solution, could you use GitHub UI / Vim to change it back?